src/main/java/pl/winiarczyk/lesson4/carExample/Car.java

```diff
@@ -8,10 +8,15 @@
     String model;
     Engine engine;
 
+    public Car() {
+        this.mileage = 25;
+    }
+
     public Car(int doors, Brand brand, String model) { //to jest konstruktor
         this.doors = doors;
         this.brand = brand;
         this.model = model;
+        this.mileage = -1000;
```

Comment:
A zmienianie tej klasy Car w tym zadanu to co to? :P Tego w ogóle nie powinno tu być

Reply:
Pozostałości po zepsutym pull requescie.